cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -114,19 +118,19 @@ func isb64(data []byte) bool {
 	return err == nil
 }
 
-func VerifyBlobCmd(ctx context.Context, ko KeyOpts, certRef, sigRef, blobRef string) error {
-	var pubKey signature.Verifier
+func VerifyBlobCmd(ctx context.Context, ko sign.KeyOpts, certRef, sigRef, blobRef string) error {
```

Comment:
```suggestion
func Blob(ctx context.Context, ko sign.KeyOpts, certRef, sigRef, blobRef string) error {
```

Reply:
:D I started doing this but all of the files are not constant. the plan is to move all of this code into another form, so this whole thing is throw away anyway soon. let's hold off renaming at this moment, I added a `// nolint` comment for now.

Reply:
ty